source/documentation/deploying_services/sqs.md

```diff
@@ -0,0 +1,343 @@
+# Amazon SQS
+
+Amazon SQS provides "Standard" (at-least-once delivery, best-effort ordering) and First-In-First-Out queues.
+
+To provide any feedback on this feature, contact the GOV.UK PaaS team at
+[gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
+
+<h2 id="set-up-the-service">Set up the service</h2>
+
+### Provision an Amazon Web Service SQS queue
+
+Creating a service instance provisions an Amazon Web Service (AWS) Simple Queue
+Service (SQS) queue.
+
+The name of the service overall is `aws-sqs-queue`. There are two types of plan
+available for the service named `standard` and `fifo`.
+
+- `standard` plans configure queues that provide at-least-once delivery,
+which means that each message is delivered at least once.
+- `fifo` (first-in-first-out) plans configure queues that provide [exactly-once
+processing](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/FIFO-queues.html#FIFO-queues-exactly-once-processing),
+which means that each message is delivered once and remains available until a
+consumer processes it and deletes it.
+
+AWS SQS queues can be configured to move messages that can not be successfully
```

Comment:
Think we should link to the section below about configuring dead-letter queue from here somewhere